server/src/interfaces.ts

```diff
@@ -61,26 +61,42 @@ export interface CommitComputedProperties {
 
 /**
  * CommitRequest Interface that contains the fields that will be provided
- * by the client in the POST/PUT request body.
+ * by the client in the request body to create a Commit.
  */
 export interface CommitRequest {
   customerId: number;
   listingId: number;
 }
 
+/**
+ * CommitPaymentRequest Interface that contains the fields that will be provided
+ * by the client in the request body.
+ */
+export interface CommitPaymentRequest {
+  deliveryAddress: string;
+  deliveryContactNumber: string; // E164 format
```

Comment:
should we have name also

Reply:
@chshda Ohh that's true, is it in case the receiver of the goods is not the customer themselves?